src/sentry/tasks/sentry_apps.py

```diff
@@ -54,24 +54,7 @@ def process_resource_change_bound(
 @instrumented_task(name="sentry.tasks.sentry_apps.installation_webhook", **CONTROL_TASK_OPTIONS)
 @retry_decorator
 def installation_webhook(installation_id: int, user_id: int, *args: Any, **kwargs: Any) -> None:
-    from sentry.sentry_apps.installations import SentryAppInstallationNotifier
-
-    extra = {"installation_id": installation_id, "user_id": user_id}
-    try:
-        # we should send the webhook for pending installations on the install event in case that's part of the workflow
-        install = SentryAppInstallation.objects.get(id=installation_id)
-    except SentryAppInstallation.DoesNotExist:
-        logger.info("installation_webhook.missing_installation", extra=extra)
-        return
-
-    user = user_service.get_user(user_id=user_id)
-    if not user:
-        logger.info("installation_webhook.missing_user", extra=extra)
-        return
-
-    SentryAppInstallationNotifier(
-        sentry_app_installation=install, user=user, action="created"
-    ).run()
+    new_installation_webhook(installation_id=installation_id, user_id=user_id, *args, **kwargs)
```

Comment:
Did this get missed in the task move changes?

Reply:
huh, I don't know why the diff looks like this, lemme check master.

Reply:
this diff is weird, master is correct with only having the `new_installation_webhook` forwarding function. I don't know why it says we removed a a bunch of code from the function. The above file is the new sentry_apps task which has the renamed Notifier and updated import, odd that they seem out of sync. I can try remerging in master again 🤷‍♀️